src/bp/core/services/module/resources-loader.ts

```diff
@@ -51,6 +62,44 @@ export class ModuleResourceLoader {
     await this._loadModuleResources()
   }
 
+  async enableResources() {
+    const ghost = this.ghost.global()
+
+    for (const path of this.globalPaths) {
+      const files = await ghost.directoryListing(path, undefined, undefined, true)
+      await Promise.all(
+        files
+          .filter(name => name.startsWith(DISABLED_PREFIX))
+          .map(file => ghost.renameFile(path, file, file.replace(DISABLED_PREFIX, '')))
+      )
+    }
+
+    const hooks = await ghost.directoryListing('/hooks', undefined, undefined, true)
```

Comment:
what about actions ?

Reply:
they are already handled just before. there's a different data structure that asks for a different method:
actions/module_name/files
hooks/hook_type/module_name/files

It was either getting a list of hooks folder and testing them individually (by appending the module name), or just fetching them all and running a regex